x-pack/plugins/actions/server/lib/task_runner_factory.ts

```diff
@@ -131,21 +136,30 @@ export class TaskRunnerFactory {
             relatedSavedObjects: validatedRelatedSavedObjects(logger, relatedSavedObjects),
           });
         } catch (e) {
-          if (e instanceof ActionTypeDisabledError) {
-            // We'll stop re-trying due to action being forbidden
-            throw new ExecutorError(e.message, {}, false);
+          logger.error(`Action '${actionId}' failed: ${e.message}`);
```

Comment:
Can we include whether the action will be retried or not in the logged message?

Reply:
Sure!